test/unit/dom/appendChild.js

```diff
@@ -38,13 +38,24 @@ describe('dom: appendChild', () => {
         }).to.throw(Error);
       });
 
-      it('should return an appended node', () => {
+      it('should return an appended div', () => {
         const appended = document.createElement('div');
         const changedElem = elem.appendChild(appended);
         expect(changedElem).not.to.equal(undefined);
         expect(changedElem).to.equal(appended);
       });
 
+      it('should return an appended document fragment', () => {
+        const appended = document.createDocumentFragment();
+        const div1 = document.createElement('div');
+        const div2 = document.createElement('div');
```

Comment:
If you testing the `return` here you really don't need to append two divs here. So maybe separate `append` test and `return` test would be a good idea?